src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.hpp

```diff
@@ -44,6 +44,27 @@ class jit_add_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
 };
 
+class jit_logical_and_emitter : public jit_emitter {
```

Comment:
Minor: may I ask you to move your implementations (`Algorithm::EltwiseLogicalAnd` and `jit_logical_and_emitter`) in the corresponding places in the files to align with alphabetical order there please? 😊